e2e/test/iothub/DeviceTokenRefreshE2ETests.cs

```diff
@@ -283,7 +283,7 @@ private class TestTokenRefresher : ClientAuthenticationWithTokenRefresh
 
             private readonly MsTestLogger _logger;
 
-            public TestTokenRefresher(string deviceId, string key, MsTestLogger logger) : base(deviceId)
+            public TestTokenRefresher(string deviceId, string key, MsTestLogger logger) : base(deviceId: deviceId)
```

Comment:
we have 2 ctor-s in `ClientAuthenticationWithSakRefresh`, one for all components (deviceId, moduleId, sharedAccessKey, sharedAcessKeyName), and the other for the whole connection string. Specifying the param name here to point to the correct ctor.